docs/agent-install-standalone-linux.md

```diff
@@ -0,0 +1,115 @@
+# Install to Linux using a GZIP File
```

Comment:
This is the standalone tar bundle.  It just happens to be gzipped.

Reply:
Fixed as follows:
Install to Linux using a tar.gz File